core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -506,4 +510,33 @@ private[spark] object UIUtils extends Logging {
 
   def getTimeZoneOffset() : Int =
     TimeZone.getDefault().getOffset(System.currentTimeMillis()) / 1000 / 60
+
+  /**
+  * Return the correct Href after checking if master is running in the
+  * reverse proxy mode or not.
+  */
+  def makeHref(proxy: Boolean, id: String, origHref: String): String = {
```

Comment:
this is not in the 2.0 PR?

Reply:
I think this method came with the original patch.  I don't see anything calling it.  I will remove it.